e2e/test/scenarios/visualizations-charts/waterfall.cy.spec.js

```diff
@@ -20,33 +22,15 @@ describe("scenarios > visualizations > waterfall", () => {
   function verifyWaterfallRendering(xLabel = null, yLabel = null) {
     // a waterfall chart is just a stacked bar chart, with 4 bars
     // (not all of them will be visible at once, but they should exist)
-    cy.findByTestId("query-visualization-root")
-      .get(".sub .chart-body")
-      .within(() => {
-        cy.get(".stack._0");
-        cy.get(".stack._1");
-        cy.get(".stack._2");
-        cy.get(".stack._3");
-      });
-    cy.findByTestId("query-visualization-root")
-      .get(".axis.x")
-      .within(() => {
-        cy.findByText("Total");
-      });
+    chartPathWithFillColor("#88BF4D");
```

Comment:
Should we do an assertion like `chartPathWithFillColor("#88BF4D").should("be.visible")`?`